shopfloor_mobile_base/static/wms/src/components/manual_select.js

```diff
@@ -115,15 +121,27 @@ Vue.component("manual-select", {
             }
         },
         handleSelect(rec, event) {
-            const elem = event.target;
+            this._updateElement(rec, event.target);
+            if (!this.opts.showActions) {
+                this._emitSelected(this._getSelected());
+            }
+        },
+        handleScan(new_data) {
+            new_data.selected_move_lines.forEach((line) => {
+                const query = $(`[input-value=${line.id}]`);
+                const elem = query[0];
+                if (line.qty_done > 0) {
```

Comment:
this cannot stay here, it's a generic component.

Reply:
Moved it out of this component.